substrate/frame/mixnet/src/lib.rs

```diff
@@ -254,7 +247,7 @@ pub mod pallet {
 		StorageDoubleMap<_, Identity, SessionIndex, Identity, AuthorityIndex, BoundedMixnodeFor<T>>;
 
 	#[pallet::genesis_config]
-	#[derive(frame_support::DefaultNoBound)]
+	#[derive(frame::prelude::DefaultNoBound)]
```

Comment:
```suggestion
	#[derive(DefaultNoBound)]
```

you use `prelude::*` above, which brings this into scope

Reply:
Good catch, thanks!